packages/exchange/tests/test_langfuse_wrapper.py

```diff
@@ -9,38 +9,40 @@ def mock_langfuse_context():
         yield mock
 
 
-@patch("exchange.langfuse_wrapper.HAS_LANGFUSE_CREDENTIALS", True)
 def test_function_is_wrapped(mock_langfuse_context):
-    mock_observe = MagicMock(side_effect=lambda *args, **kwargs: lambda fn: fn)
-    mock_langfuse_context.observe = mock_observe
+    with patch("exchange.langfuse_wrapper.auth_check") as mocked_auth_check:
```

Comment:
why this approach? i deliberately wrote the tests as a decorator to reduce how much nesting the testing code has